docs/src/using-turing/guide.md

```diff
@@ -545,6 +545,19 @@ The `Gibbs` sampler can be used to specify unique automatic differentation backe
 
 For more details of compositional sampling in Turing.jl, please check the corresponding [paper](http://proceedings.mlr.press/v84/ge18b.html).
 
+### Indexed Group Variables
+
+The function `filldist(distribution,n)` provides a simplified interface for defining a set of model variables that share the same structure, but vary by group.
```

Comment:
Maybe this could be formulated in a different way? IMO it sounded a bit like `filldist` is designed specifically for this purpose, even though that's not the case and it is just a general way of creating a product distribution, in which all single distributions are the same and that allows also multivariate distributions (in contrast to `Product` in Distributions).

It would be good to mention `arraydist` as well in case they not all share the same structure.

Maybe something like, "if you want to model a set of variables that share the same structure but vary by group, you can use `filldist` and `arraydist`."

Reply:
Yes, I agree. Maybe there could be two examples, one with and one without `filldist` or `arraydist`?